### PR TITLE
[Website] Fixed `footer` items alignment

### PR DESCRIPTION
**Pull request by NikhilSharma03**
Signed-off-by: Nikhil Sharma <nikhilsharmamusic2000@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

### Before Changes
Screenshot from 2022-01-02 16-52-59

### After Changes
Screenshot from 2022-01-02 16-59-45

### CODE REVIEW

**Review by wiardvanrij — APPROVED** [review]
Nice work!

File: website/static/main.css [hunk 1]
```diff
@@ -179,9 +179,17 @@ input[type="search"]::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
     }
 }
 
-@media (max-width: 768px) {
+@media (max-width: 767px) {
     .hero-logo-container {
         flex-grow: 1;
         max-width: 100%;
     }
+    .footer-container {
```
Comment (squat) [inline website/static/main.css:187]: Tiny nit:
```suggestion

    .footer-container {
```

Comment (NikhilSharma03) [reply website/static/main.css:188]: done

**Review by kakkoyun — APPROVED** [review]
LGTM

**Review by squat — APPROVED** [review]
thanks :)

**Pull request merged**